Sources/TuistGenerator/Linter/GraphLinter.swift

```diff
@@ -5,7 +5,7 @@ import TuistGraph
 import TuistSupport
 
 public protocol GraphLinting: AnyObject {
-    func lint(graphTraverser: GraphTraversing) -> [LintingIssue]
+    func lint(graphTraverser: GraphTraversing, disableStaticProductsLint: Bool) -> [LintingIssue]
 }
```

Comment:
Seeing this is a workspace generation option, perhaps can be extracted from `graphTraverser.workspace`?

This should reduce the necessary code changes 😅